### PR TITLE
Builds are now in release mode, purged unnecessary fluff from the sou…

### PR DESCRIPTION
**Pull request by JonathanHenson**
…rce distribution. Build output is now 800kb, and source distro is less than 4.5 MB.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,57 @@ Python bindings for the AWS Common Runtime
...
+
+````bash
+<path to your python binary> -m twine upload --repository-url <the repo> dist/*
+````
+
+## Using From Your Python Application
+
+Normally, you just declare `aws_crt` as a dependency in your setup.py file.
+
+### Installing from pip
```
Comment (justinboswell) [inline README.md]: Shouldn't this come first, as it's the most user-friendly?

File: setup.py [hunk 2]
```diff
@@ -212,7 +213,7 @@ def build_dependency(lib_name):
 
 setuptools.setup(
     name="aws_crt",
-    version="0.0.1",
+    version="0.2.2",
```
Comment (justinboswell) [inline setup.py]: Shouldn't this be 0.2.0?

Comment (JonathanHenson) [reply setup.py]: no, it should match the current tag, which will be cut as soon as I merge this.

File: README.md (hunk 1 above)
Comment (JonathanHenson) [reply README.md]: er.... this isn't really for them, and publish then consume seems most logical for the layout. i just don't want the knowledge of how to publish this stuff to get lost in my head or on my bash history.

Comment (justinboswell) [reply README.md]: That's fair, though maybe it should go in PACKAGING.md? :)

Either way, I'm not super particular, it just jumped out at me.

File: setup.py (hunk 2 above)
Comment (justinboswell) [reply setup.py]: I should have looked, no worries.

**Pull request merged**